storage/series/series_mock.go

```diff
@@ -26,14 +26,13 @@ package series
 import (
 	time "time"
 
+	gomock "github.com/golang/mock/gomock"
 	context "github.com/m3db/m3db/context"
 	persist "github.com/m3db/m3db/persist"
 	block "github.com/m3db/m3db/storage/block"
 	ts "github.com/m3db/m3db/ts"
 	io "github.com/m3db/m3db/x/io"
 	time0 "github.com/m3db/m3x/time"
```

Comment:
I had to edit this generated mock by hand. For some reason, it was trying to import itself.